src/main/java/net/ccbluex/liquidbounce/features/module/Module.kt

```diff
@@ -47,9 +51,6 @@ open class Module @JvmOverloads constructor(
             saveConfig(modulesConfig)
         }
 
-    val description: String
-        get() = forcedDescription.ifBlank { translation("module.${name.toLowerCamelCase()}.description") }
```

Comment:
This is the correct way to do it.

Reply:
Should be resolved https://github.com/CCBlueX/LiquidBounce/pull/1001/commits/4270f255cc86e43802b5e3d159070c78f132d5b0